docs/network/network-binding-plugin.md

```diff
@@ -505,3 +505,10 @@ derived from virtual machines using the plugin.
 For a network binding plugin to support compute resource overhead, the `computeResourceOverhead` field
 must be specified in the Kubevirt CR.
 See the user-guide network binding plugin [section](https://kubevirt.io/user-guide/network/network_binding_plugins/#register) on how to define it.
+
+## Network plugin custom sockets
```

Comment:
Please consider adjusting the wording to emphasize that KubeVirt scans the <dir_name> directory for Unix sockets to communicate with the sidecar.
This directory should not contain directly contain sockets used for other purposes.